apps/hub-app/src/pages/HomePage.tsx

```diff
@@ -85,11 +86,20 @@ const HomePage = () => {
       setLoading(true);
       try {
         const haus = temporaryInitHaus();
+
```

Comment:
imo, getting this query is a more separate concern than making the fetch (intention of this fn), I would refactor this into its own function. 

Would also remove variable assignment to remove extra moving parts

Reply:
Just saw the delegate data. That's helpful. Thanks!

Reply:
@jordanlesich 

take a look at the refactor at your convenience.